keyboards/crkbd/keymaps/ardakilic/keymap.c

```diff
@@ -0,0 +1,482 @@
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    |        |        |        |        |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+                                    ,-----------------------,  ,-----------------------.
+                                    |       |       |       |  |       |       |       |
+                                   /|-------+-------+-------'  +-------+-------+-------|\
+  ),
+*/
+  [_SODA] = LAYOUT_split_3x6_3(
+    _______,  _______,  KC_UP,  _______,  _______,  _______,        _______, _______, _______, _______, _______, _______,
+    _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,  _______,  _______,      _______, _______, _______, _______, _______, _______,
+    _______,  _______,  _______,  _______,  _______,  _______,      _______, _______, _______, _______, _______, _______,
+                                      _______, _______, _______,    _______, _______, _______
+  ),
+
+
+
+/*
+ * ADJUST LAYER
+  [_TPL] = LAYOUT_wrapper( \
+  //,-----------------------------------------------------.      ,----------------------------------------------------.
+    |  RESET | EEPRST |        |        |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    | RGBTog |  HUE▲  |  SAT▲  | BRGHT▲ |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+    | RGBMOD |  HUE▼  |  SAt▼  | BRGHT▼ |        |        |      |        |        |        |        |        |       |
+  //|--------+--------+--------+--------+--------+--------|      |--------+--------+--------+--------+--------+-------|
+                                    ,-----------------------,  ,-----------------------.
+                                    |   OS  |       | Space |  |  Ent  |       |  ALT  |
+                                   /|-------+-------+-------'  +-------+-------+-------|\
+  ),
+*/
+  [_ADJUST] = LAYOUT_split_3x6_3(
+    RESET, EEP_RST, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______,                    _______, _______, _______, _______, _______, _______,
+    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______,                    _______, _______, _______, _______, _______, _______,
+                                          KC_LGUI, _______,  KC_SPC,     KC_ENT, _______, KC_RALT
+  )
+};
+
+
+#ifdef OLED_DRIVER_ENABLE
+#include <stdio.h>
```

Comment:
```suggestion
```

Just a heads up, snprintf/sprintf uses up 1.5kb of firmware space, compared to something like this: 
```c
    uint8_t n = get_current_wpm();
    char wpm_counter[4];
    wpm_counter[3] = '\0';
    wpm_counter[2] = '0' + n % 10;
    wpm_counter[1] = ( n /= 10) % 10 ? '0' + (n) % 10 : (n / 10) % 10 ? '0' : ' ';
    wpm_counter[0] = n / 10 ? '0' + n / 10 : ' ';
    oled_write_P(PSTR("WPM: "), false);
    oled_write(wpm_counter, false);
```

Reply:
Thanks @drashna , this is awesome!